### ISSUE TITLE
When logging is disabled via command line, log level drop down should be greyed out

### ISSUE THREAD

**Issue by Qchristensen** [nvaccess/nvda#10209]
### Steps to reproduce:

Start NVDA either with the --no-logging, --secure or --log-level=100 command line switches

Open the General settings dialog (NVDA+control+g)
TAB to the Log Level drop-down
Modify the value

### Actual behavior:

The drop-down selection can be changed, although the logging level is not in fact changed when the apply or OK buttons are activated.

### Expected behavior:

The log level drop-down should be greyed out so the user cannot interact with it.  I've already had contact from one IT administrator trying to setup NVDA securely on their system, concerned that although they have disabled logging, this drop-down can be changed, which gives the appearance that the log level can be changed.

#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
2019.2
#### Windows version:
Windows 10 Version: 
18362.1 
#### Does the issue still occur after restarting your PC?
Yes

**Comment by feerrenrut** [comment]
If someone wants to look at this, the change will need to be made in `source/gui/settingsDialogs.py`  in the `makeSettings` method of `GeneralSettingsPanel`. [Disable ](https://wxpython.org/Phoenix/docs/html/wx.Window.html#wx.Window.Disable) `self.logLevelList` when the log level is set from the command line. Probably best to extract the logic at the start of the `setLogLevelFromConfig` function in `source/logHandler.py`, put it in a new function in `source/logHandler.py` and use that.  It looks like there may be several other parts of the code base that rely on this same condition, it might be nice to centralise those while here.
